### PR TITLE
Clarified docs to address developer questions

### PR DESCRIPTION
**Pull request by maudnals**
- Clarified the section on redirects to address incoming developer question
- Added short deeplinks

### CODE REVIEW

**Comment by maudnals** [comment]
This is ready to merge. Thanks!

**Comment by jpmedley** [comment]
Thank you for your submission. The reviewer with the most expertise in this area will be back in the office next week. Please bear us.

File: site/en/docs/privacy-sandbox/attribution-reporting-changes-january-2022/index.md [hunk 1]
```diff
@@ -19,7 +19,12 @@ under **Join the public discussion** in this post, or by creating new issues.
 [Learn more about participating](/docs/privacy-sandbox/attribution-reporting-introduction/#participate).
 {% endAside %}
 
-## Who is this post for?
+## Changelog to this article
+
+- 7/2/2022: Added section on [header trigger redirect](#header-trigger-redirect)
+- 27/1/2022: Article first published
```
Comment (mihajlija) [inline site/en/docs/privacy-sandbox/attribution-reporting-changes-january-2022/index.md:25]:
```suggestion
- February 7, 2022: Section on [header trigger redirect](#header-trigger-redirect) added.
- January 27, 2022: Article first published.
```

**Review by mihajlija — APPROVED** [review]
LGTM

**Comment by maudnals** [comment]
Thank you!

**Pull request merged**